app/Console/Commands/AddClient.php

```diff
@@ -72,6 +75,25 @@ private function addClient(ClientRepository $clientRepository, string $name)
         return $client->id;
     }
     
+    /**
+     * Encrypt and store the webling api key in the WeblingKey table
+     *
+     * @param int $clientId
+     * @param string $key
+     */
+    private function addWeblingKey(int $clientId, string $key)
+    {
+        $keyModel = new \App\WeblingKey();
+        $keyModel->api_key = Crypt::encryptString($key);
```

Comment:
Müsste man dieses Crypt nicht noch irgendwie konfigurieren?
Also konkret einen Key generieren und in config/app.php ablegen - siehe: https://laravel.com/docs/5.7/encryption

... Oder macht man das erst auf der Instanz? (Natürlich auch nicht sinnvoll, wenn der Key hier in der Repo zu sehen ist...)

Reply:
Grundsätzlich scheint es mir sinnvoll, wenn jede Instanz ihre eigenen Keys hat. Sobald keys geteilt werden, erhöht sich das Risiko eines Leaks.

Der Key wird durch `php artisan key:generate` erzeugt und ins .env geschrieben. Wenn laravel über den composer installiert wird, passiert dies automatisch.